tests/unit/ext/switches/brocade.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2016 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+import requests_mock
```

Comment:
Might be good to more clearly separate rigging up the responses from making the calls; fixtures? For some of these it looks like they're only used the once, in which case a comment and a blank line might be sufficient.